client.go

```diff
@@ -63,6 +63,32 @@ func (c *Client) EasyGet(url string, opts *Options) (*Response, error) {
 	}, nil
 }
 
+
+// EasyHead make request and returns response
```

Comment:
```
// EasyHead make head request
// It accept header values as Options and returns extended Response that acts as *http.Response
```
this could be more clear